test-data/unit/check-newsemanal.test

```diff
@@ -2716,3 +2716,34 @@ class N(NamedTuple):
         )
         b
 [builtins fixtures/tuple.pyi]
+
+[case testNewAnalyzerLessErrorsNeedAnnotation]
```

Comment:
What about adding a test case for `x = []  # type: ignore` followed by `reveal_type(x)`?